app/res/values-pt/strings.xml

```diff
@@ -523,4 +523,10 @@
     <string name="recovery_network_cooldown">Demasiadas tentativas de pedido. Por favor, tente novamente dentro de uma hora.</string>
     <string name="personalid_authentication_failed">A autenticação da biometria no seu dispositivo Android falhou. Por favor, aplique quaisquer atualizações de sistema disponíveis ou tente novamente com um dispositivo diferente.</string>
     <string name="no_channel_msg">Você não possui canais de mensagens ativos.</string>
+
+    <string name="personalid_using_your_location">Usando sua localização</string>
+    <string name="personalid_no_location_found">Não conseguimos encontrar a sua localização</string>
+    <string name="personalid_tooltip_location_failure_message">Usamos dados de localização para garantir o uso seguro deste aplicativo.\nPara continuar, ative as permissões de localização nas configurações ou tente novamente mais tarde.</string>
+    <string name="personalid_tooltip_location_success_message">Usamos dados de localização para garantir o uso seguro deste aplicativo.\n<a href="https://www.dimagi.com/terms/latest/privacy/">Clique aqui para mais informações.</a></string>
```

Comment:
Nit: We should add another newline in-between the sentences for better readability for the user.

```suggestion
    <string name="personalid_tooltip_location_failure_message">Usamos dados de localização para garantir o uso seguro deste aplicativo.\n\nPara continuar, ative as permissões de localização nas configurações ou tente novamente mais tarde.</string>
    <string name="personalid_tooltip_location_success_message">Usamos dados de localização para garantir o uso seguro deste aplicativo.\n\n<a href="https://www.dimagi.com/terms/latest/privacy/">Clique aqui para mais informações.</a></string>
```